src/prompt.hpp

```diff
@@ -0,0 +1,13 @@
+//
+//  prompt.hpp
+//
+
+#ifndef prompt_hpp
+#define prompt_hpp
+
+#include <stdio.h>
```

Comment:
Can this include be moved to the cpp file?

Reply:
I understand your question to mean that the `#include <stdio.h>` in prompt.hpp is required.
Of course it is not. Xcode generates me a header template that contains this include  and I forgot to just delete this line. stdio.h.
The `stdio.h` is also not needed in prompt.cpp, so it is doubly redundant here.